ci-operator/step-registry/openshift/e2e/gcp/libvirt/conf/openshift-e2e-gcp-libvirt-conf-commands.sh

```diff
@@ -14,12 +14,18 @@ cat > "${SHARED_DIR}"/test-list << EOF
 [Conformance][Area:Networking][Feature:Router] The HAProxy router should run even if it has no access to update status [Suite:openshift/conformance/parallel/minimal]
 [Conformance][templates] templateinstance creation with invalid object reports error  should report a failure on creation [Suite:openshift/conformance/parallel/minimal]
 [Conformance][templates] templateinstance cross-namespace test should create and delete objects across namespaces [Suite:openshift/conformance/parallel/minimal]
+[sig-devex][Feature:Templates] templateinstance cross-namespace test should create and delete objects across namespaces [apigroup:user.openshift.io][apigroup:template.openshift.io] [Suite:openshift/conformance/parallel]
```

Comment:
All the duplicated tests in this file got renamed through https://github.com/openshift/origin/pull/24683 more than 2 years ago (Mar 13, 2020). CC @praveenkumar @sallyom 

Re-enabling them might cause new failing tests appear.

Reply:
@ingvagabund if e2e-libvirt passes with this change then we shouldn't have any issue.

Reply:
looks like `resharse` for `e2e-libvirt` will not run because change  in command.sh not job template so this need a manual test.

Reply:
@praveenkumar chicken egg problem. Changes introducing the apigroups for the affected tests got reverted in https://github.com/openshift/origin/pull/27378. I need this change merged first so I can revert https://github.com/openshift/origin/pull/27378.